chapters/zh/chapter1/8.mdx

```diff
@@ -0,0 +1,31 @@
+# Bias and limitations
```

Comment:
While looking at the rendered output I noticed this is still in English. Would it make sense to translate it too?

Reply:
Perfect! I'll merge this now :)